explain/grid.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: 'The pganalyze EXPLAIN grid view'
+backlink_href: /docs/explain
+backlink_title: 'Documentation: EXPLAIN - Overview'
+---
+
+import imgGrid from './grid.png'
+
+export const ImgGrid = () => <img src={imgGrid} alt="Plan Grid Example" />
+
+The pganalyze EXPLAIN grid view is an alternate way to inspect query plans. It's
+more compact, and modeled after the native Postgres text EXPLAIN format, but
+includes some quality of life improvements.
+
+<ImgGrid />
+
+Note that this view is a work in progress, and may not include all the plan
+information you're interested in yet. In that case, check the Node Tree EXPLAIN
+view or the source of individual nodes (and please [contact us with
+feedback](mailto:support@pganalyze.com)).
+
+## Plan layout
+
+The query plan is displayed in a way similar to the Postgres text EXPLAIN format,
+but with some important differences:
+
+ - Common Table Expressions (WITH clause entries) and any other independent
+   subqueries are broken out into their own section.
+ - The focus is on the shape of the plan and node details are available only
+   after clicking to select a specific node.
+ - Scan details include the table being scanned, the index used (if any), and
+   the alias used for this table reference in the query (if any).
+
+
+## Metrics
+
+The execution metric to show can be selected at the top of the plan:
+
+  - **Est. Cost**: The cost of the node according to the Postgres planning model,
+    excluding the cost of the node's children.
+  - **Runtime**: How much time was spent executing this node, excluding time spent
+    executing the node's children.
+  - **Rows**: How many rows each node produced, and how many it was expected to
+    produce according to the query plan
```

Comment:
Looks like some lines have periods at the end and some don't, would be good to unify

Reply:
Ah, I tried to, but I guess I missed some. Thanks!